packages/components/src/input-control/test/index.js

```diff
@@ -65,6 +65,7 @@ describe( 'InputControl', () => {
 			const help = screen.getByRole( 'link', { name: 'My help text' } );
 
 			expect(
+				// eslint-disable-next-line testing-library/no-node-access
```

Comment:
This will have to be intentionally ignored, unless RTL gets better support for `aria-details`.